main.go

```diff
@@ -9,14 +10,14 @@ import (
 	"github.com/urfave/cli"
 )
 
-var version = "1.0.0" // build number set at compile-time
+var build = "0" // build number set at compile-time
 
 func main() {
 	app := cli.NewApp()
 	app.Name = "volume cache plugin"
 	app.Usage = "valume cache plugin"
 	app.Action = run
-	app.Version = version
+	app.Version = fmt.Sprintf("1.0.%s", build)
```

Comment:
maybe `1.0.0+%s` based on the semver examples?

> Thus two versions that differ only in the build metadata, have the same precedence. Examples: 1.0.0-alpha+001, 1.0.0+20130313144700, 1.0.0-beta+exp.sha.5114f85.

Reply:
Sure, I just took this from other plugins